src/lib/modernExtend.ts

```diff
@@ -483,3 +483,34 @@ export function forceDeviceType(args: {type: 'EndDevice' | 'Router'}): ModernExt
     };
     return {configure, isModernExtend: true};
 }
+
+// XXX: add options
+export function temperature(args?: Partial<NumericArgs>) {
+    return numeric({
+        name: 'temperature',
+        cluster: 'msTemperatureMeasurement',
+        attribute: 'measuredValue',
+        configureReporting: {minimumReportInterval: 10, maximumReportInterval: repInterval.HOUR, reportableChange: 100},
```

Comment:
I took the same values as in src/lib/reporting.ts

Reply:
I think you also need ` scale: 100,` here?

Reply:
Yes, It's already fixed locally. I'll rebase and push, still working on the moving over the calibration code.